cnames_active.js

```diff
@@ -2977,6 +2977,7 @@ var cnames_active = {
   "suggestcord": "suggestcord.github.io/docs",
   "suggester": "suggester.github.io/Documentation",
   "suka": "sukkaw.github.io",
+  "sulin": "https://alltobebetter.github.io/sulinjs/"
```

Comment:
```suggestion
  "sulin": "alltobebetter.github.io/sulinjs",
```
As the CI error notes:

1. You do not need `https://`
2. You do not need the trailing `/`
3. You need a comma at the end of the line